libs/blocks/chart/chart.js

```diff
@@ -116,28 +150,43 @@ export const getChartOptions = (chartType, dataset, colors, unit = '') => {
 };
 
 const init = async (el) => {
+  const parent = el.parentElement;
   const children = el?.querySelectorAll(':scope > div');
   children[0]?.classList.add('chart_title');
   children[1]?.classList.add('chart_subTitle');
   children[3]?.classList.add('chart_footnote');
 
-  // ToDo: Replace hardcoded size MWPW-112994
-  const size = LARGE;
+  // Add chart container
+  const container = document.createElement('div');
+  container.className = 'chart_container';
+  container.append(...children);
+  el.appendChild(container);
+
+  // Grab section metadata
+  const chartStyles = parent.className.split(' ').slice(1);
+  parent.classList.remove(...chartStyles);
+
+  const size = chartStyles?.find(style => style === SMALL || style === MEDIUM || style === LARGE);
+  el.classList.add(`chart_${size}`);
+
+  const chartType = chartTypes?.find((type) => el?.className?.indexOf(type));
+  el.classList.remove(chartType);
+
   const chartRow = children[2];
+  const containerSize = getContainerSize(size, chartType);
 
   if (chartRow) {
-    chartRow.style.width = '600px';
-    chartRow.style.height = '600px';
+    chartRow.style.width = containerSize?.width;
+    chartRow.style.height = `${containerSize?.height}px`;
```

Comment:
Very minor, but why not put the height as a string including px in the object above, unless we do any mathematical comparisons against height?

Reply:
That's a good point, have removed the `else` statement. Also, I was thinking about including px in the object above but thought it was easier to just add in as a template literal instead. I don't believe we do any mathematical comparisons against height though.

Reply:
Totally, I suppose why it is standing out to me is because you are accessing two properties that one would assume would be numbers from the same object and they have two different types since you have to use a template literal for the later. Begs the question why not `'${containerSize?.width}%'`; 

I am being super nit picky here, so no worries if you don't want to change.

Also, since the width is always the same, 100%, could you store it somewhere else, like:
```
const containerSizes = {
      width: '100%',
      area: {
          small: { height: 345},
          medium: { height: 310},
          large: { height: 512},
      },
      ...
```
could even flatten the data more by making it something like:
```
area: {
          smallHeight: 345,
          mediumHeight: 310,
          largeHeight: 512,
      },
```

Reply:
Thanks, definitely some good points. You're right, restructuring the object could be beneficial here. I'd like to get this PR merged in to allow Megan to continue with her resizing work, but can make an update to this in the next PR I push up.